scaaml/capture/scope/lecroy/lecroy.py

```diff
@@ -133,32 +196,58 @@ def __init__(self, samples: int, offset: int, ip_address: str,
         self._trigger_channel = trigger_channel
         self._communication_timeout = communication_timeout
         self._trigger_timeout = trigger_timeout
+        self._communication_class_name = communication_class_name
 
         # Trace and trigger
         self._last_trace = None
 
         # Scope object
         self._scope_communication: Optional[LeCroyCommunication] = None
 
+        # Desired settings of the scope
+        scope_setup_commands = deepcopy(scope_setup_commands)
+        # Wrap default commands around the custom ones
+        scope_setup_commands = [
```

Comment:
Change the order for a more natural way instead of re-defining the variable a few times:
```python
commands = [
   ...
]
commands.extend(deepcopy(scope_setup_commands))
commands.extend([
   ...
])
self._scope_setup_commands = tuple(commands)
```